joomla/joomla-cms
# [4.0] [com_menus] Untranslated string

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/29001.

### Summary of Changes

Restores a string that was removed but is still in use.

### Testing Instructions

> Install Blog Sample Data.
> Go to Menus > All Menu Items
> Edit Home.
> Click the browser's back button so that the Home menu item is checked out.
> Click the Home icon of another menu item to make it the default.
> See error/success alerts.

### Expected result

No untranslated strings.

### Actual result

Untranslated `COM_MENUS_ITEMS_SET_HOME` string.

### Documentation Changes Required

No.
